models/actions/permissions.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2025 The Gitea Authors. All rights reserved.
```

Comment:
It's better to move this file to act or act_runner or at least `modules/actions`

Reply:
👍🏻 , thanks! Will do